licenses/tests/test_utils.py

```diff
@@ -152,3 +157,99 @@ def test_get_license_url_from_legalcode_url(self):
             get_license_url_from_legalcode_url(
                 "http://opensource.org/licences/bsd-license.php"
             )
+
+
+class GetLicenseUtilityTest(TestCase):
+    """Test django-distill utility functions for
+    generating an iterable of license dictionaries
+    """
+
```

Comment:
With CC-8, we'll no longer have licenses in the database when tests start. Which is probably just as well. We'll need to create the licenses we want for each test using e.g. LicenseFactory.

(I don't think these tests will fail, but they won't really be testing anything if there are no licenses.)

Reply:
Imported `LicenseFactory` and added licenses to `GetLicenseUtilityTest`.